### ISSUE TITLE
k8's nodes not ready after installation using Kubernetes Version 1.16

### ISSUE THREAD

**Issue by jmolmo** [galexrt/k8s-vagrant-multi-node#52]
**Bug Report**
k8's nodes not ready after installation using Kubernetes Version 1.16.

It seems that there is any kind of problem with the cni installation, but im not able to locate what can be the source of the problem. 
Any help with this will be welcomed, i'm available to investigate more or to test the possible solution.
Thanks!

**Expected behavior:**
Install a kubernetes functional kubernetes cluster version 1.16.0

**Deviation from expected behavior:**
Using the last version of k8, vms installed but k8 nodes never reach the "ready" state

**How to reproduce it (minimal and precise):**

Execute: 
NODE_COUNT=1 make up -j4

**Environment**:
* OS of the machine (e.g. from `/etc/os-release`): Fedora release 28
* Kernel of the machine (e.g. `uname -a`): Linux juanmipc 5.0.5-100.fc28.x86_64 #1 SMP Wed Mar 27 22:16:29 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
* `make versions` output: 

```
=== BEGIN Version Info ===
Repo state: 6b4302ba3d6aa4c127267746834238cb815ae5cd (dirty? YES)
make: /usr/bin/make
kubectl: /usr/local/bin/kubectl
grep: /usr/bin/grep
cut: /usr/bin/cut
rsync: /usr/bin/rsync
openssl: /usr/bin/openssl
/dev/urandom: OK
Vagrant version:
Vagrant 2.2.5
vboxmanage version:
6.0.4r128413
=== END Version Info ===
```

Other useful informations:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2018-10-10T16:38:01Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:27:17Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```

```
$ kubectl get nodes
NAME      STATUS     ROLES     AGE       VERSION
master    NotReady   master    8m30s     v1.16.0
node1     NotReady   <none>    7m48s     v1.16.0
node2     NotReady   <none>    7m48s     v1.16.0
node3     NotReady   <none>    7m46s     v1.16.0


$ kubectl describe master
...
Conditions:
  Type             Status  LastHeartbeatTime                 LastTransitionTime                Reason                       Message
  ----             ------  -----------------                 ------------------                ------                       -------
  MemoryPressure   False   Fri, 20 Sep 2019 09:33:00 +0200   Fri, 20 Sep 2019 09:32:31 +0200   KubeletHasSufficientMemory   kubelet has sufficient memory available
  DiskPressure     False   Fri, 20 Sep 2019 09:33:00 +0200   Fri, 20 Sep 2019 09:32:31 +0200   KubeletHasNoDiskPressure     kubelet has no disk pressure
  PIDPressure      False   Fri, 20 Sep 2019 09:33:00 +0200   Fri, 20 Sep 2019 09:32:31 +0200   KubeletHasSufficientPID      kubelet has sufficient PID available
  Ready            False   Fri, 20 Sep 2019 09:33:00 +0200   Fri, 20 Sep 2019 09:32:31 +0200   KubeletNotReady              runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:docker: network plugin is not ready: cni config uninitialized
...
```

**Comment by galexrt** [comment]
@jmolmo Please pull the latest `master` and try again.
Commit fbd4022 should have fixed the CNI issue(s) on v1.16, by updating the URLs to the Canal and Calico manifests.

**Comment by jmolmo** [comment]
> @jmolmo Please pull the latest `master` and try again.
> Commit [fbd4022](https://github.com/galexrt/k8s-vagrant-multi-node/commit/fbd40224e3ee8319a655dbd30858e38e105f3fb5) should have fixed the CNI issue(s) on v1.16, by updating the URLs to the Canal and Calico manifests.

It worked perfectly. Thank you very much!
